src/Components/test/E2ETest/ServerExecutionTests/ServerSideAppTest.cs

```diff
@@ -25,10 +26,12 @@ public class ServerSideAppTest : ServerTestBase<AspNetSiteServerFixture>
             _serverFixture.BuildWebHostMethod = ComponentsApp.Server.Program.BuildWebHost;
         }
 
-        protected override void InitializeAsyncCore()
+
+        public override async Task InitializeAsync()
         {
+            await base.InitializeAsync();
             Navigate("/", noReload: false);
-            WaitUntilLoaded();
+            await Task.Delay(2000);
```

Comment:
This seems like a step backwards. Why aren't we able to use the `Browser.*` assertions to wait just until the page has loaded, instead of having a hardcoded delay time?

Reply:
We don't have a visible marker that tells us the page has successfully loaded. Once @rynowak PR is in we can improve this.

Reply:
Updated it to check for a message on the log